<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <h1>Curriculum Vitae of Yirui Tang, PhD.</h1>

    <section>
        <h2>PROFILE</h2>
        <p>Bioinformatic PhD. specializing in identification of novel associations and causations between microbiome function and taxonomy with various biological variables, eg. treatment, metabolomic data, health characteristics, dietary intake data.</p>
        <p>Proficient in a broad range of bioinformatic, machine learning, and data analysis tools including Python, Bash, Git, R, Snakemake, Nextflow, Slurm, Bowtie, BMTagger, Trimmomatic, HuMann3, DESeq2, random forest, LASSO, etc.</p>
    </section>

    <section>
        <h2>EDUCATION</h2>
        
    </section>

    <section>
        <h2>GENERAL WORK EXPERIENCE</h2>

        <section>
            <span>Graduate Student Researcher, RENEW Project</span>
            <span>Western Human Nutrition Research Center, CA and UC Davis, Davis, CA</span>
            <ul>
                <li>Conduct non-hypothesis driven machine learning analysis (TaxaHFE, random forest, lasso, ridge) to predict changes in calcium absorption and bone mineral density (BMD) using metadata, clinical health characteristics, dietary intake, microbial taxa and function, faecal pH, faecal metabolites, urine metabolites, and supplement intervention.</li>
                <li>Construct metagenomic workflow pipeline and preprocess and process human fecal shotgun metagenomic sequencing data using programming engine Snakemake and software such as Metaphlan4, HuMann3, Diamond, and MicrobeCensus; statistical analysis of metagenomic data using Phyloseq, DESeq2, MaAsLin3, and ANCOVA.</li>
                <li>Custom databases for microbial genes involved in lactose and inulin utilization.</li>
                <li>Perform data cleaning and quality control for subjects’ metadata and dietary intake data.</li>
                <li>Determine the effect of different sources of calcium supplement and prebiotics (inulin) on the gut microbiome taxa and function.</li>
                <li>Submitted 800 fecal samples comprising over 3,000 shotgun metagenomic sequencing files to the NCBI Sequence Read Archive (SRA).</li>
                <li>Collaborate with multiple microbiology, metabolomics, and bone health research and clinic laboratories to secure data sharing and result communication.</li>
                <li>Communicate research results with peer scientists in regular meetings and oral and poster sessions in nutrition conference, such as American Society for Nutrition (ASN) 2025 and microbiology conference, such as American Society for Microbiology (ASM) 2025.</li>
            </ul>
        </section>

        <section>
            <span>Graduate Student Researcher, CDRF Project</span>
            <span>Western Human Nutrition Research Center, CA and UC Davis, Davis, CA</span>
            <ul>
                <li>Analyze human fecal short-chain fatty acid levels in relation to lactase-persistent / lactase-nonpersistent phenotype, as well as to lactose intake, dairy intake, various dairy product intake, and fecal microbiome composition and abundance.</li>
                <li>Construct metagenomic workflow pipeline and preprocess human fecal shotgun metagenomic sequencing data using Metaphlan4, HuMann3, Diamond, and MicrobeCensus; statistical analysis of metagenomic data using DESeq2, MaAsLin, etc.</li>
                <li>Communicate research results with peer scientists in regular meetings and poster sessions in nutrition conference, such as American Society for Nutrition (ASN) 2023 and the Graduate Group of Nutritional Biology (GGNB) Student Research Symposium 2023.</li>
                <li>Custom bacterial beta-galactosidase gene database.</li>
                <li>Custom pathway databases for microbial genes involved in lactose degradation.</li>
            </ul>
        </section>

        <section>
            <span>Graduate Student Researcher, DHB project</span>
            <span>Western Human Nutrition Research Center, CA and UC Davis, Davis, CA</span>
            <ul>
                <li>Develop a Dairy Health Benchmark (DHB) database that consists of all known molecules (over 10,000 entries) in bovine milk by reviewing all relevant literatures and relevant existing databases; excel and R were used to process the datasets.</li>
                <li>Curate molecule-health relationships for selected bovine milk molecules using SPLASH and PubMed API; python, bash script, and xml were used extensively.</li>
                <li>Continue developing the Dairy Health Benchmark (DHB) database.</li>
                <li>Deduplicate DHB database based on Pubchem IDs and synonyms of molecular names.</li>
                <li>Communicate DHB database with peer scientists in regular meetings and poster sessions.</li>
            </ul>
        </section>
        
        <section>
            <span>Graduate Student Researcher</span>
            <ul>
                <li>Preprocess 18 samples of human gut microbiome metagenome for functional and taxonomy analysis.</li>
                <li>Statistical analysis and graph visualizations of antimicrobial resistant genes (ARGs) in the new human gut microbiome results.</li>
                <li>Contribute significantly on methods section in the manuscript.</li>
                <li>Create scripts for metagenome-corrected metatranscriptomes analysis.</li>
                <li>Submitted 18 fecal samples of shotgun metagenomic sequencing files to the NCBI SRA.</li>
            </ul>
        </section>

        <section>
            <span>Junior Specialist</span>
            <ul>
                <li>Create python and bash scripts for customized human gut microbiome metagenomic analysis pipeline. The pipeline includes, but not limited to, BMTagger, Trimmomatic, FastUniq, FLASH, DIAMOND, MicrobeCensus, MEGAR, MEGAHIT, BWA alignment, Kraken2, Bracken.</li>
                <li>Statistical analysis and graph visualizations of antimicrobial resistant genes (ARGs) in the human gut microbiome results.</li>
                <li>Review literature on antimicrobial resistant genes (ARGs) study in animals or human models.</li>
                <li>Contribute on abstract, background, methods, and result sections in the manuscript.</li>
            </ul>
        </section>

        <section>
            <span>Laboratory Internship</span>
            <ul>
                <li>Review literature and existing databases to collect microbial genes involved in B vitamin synthesis and production.</li>
                <li>Create customized database for desired genes in the form of a fasta file for further analysis of microbial capacity of synthesizing and utilizing B vitamins.</li>
            </ul>
        </section>

        <section>
            <span>Volunteer Internship</span>
            <ul>
                <li>Develop highly efficient drug-repurposing methods by targeting fungal antioxidant system.</li>
                <li>Apply chemosensitization strategy that utilizes redox-active natural compounds or their structural analogs as chemosensitizers.</li>
                <li>Use fungal mutants that lack key genes in antioxidant system, which previously have resulted in enhanced efficacy of repurposed pro-oxidant drug—bithionol in filamentous fungus, Aspergillus fumigatus.</li>
                <li>Results from the model yeast Saccharomyces cerevisiae indicated that the sensitivity of drug-repurposing process in S. cerevisiae did not follow that observed in A. fumigatus, thus suggesting strain-specificity in the drug-repurposing developed.</li>
            </ul>
        </section>

        <section>
            <span>Volunteer Laboratory Internship </span>
            <ul>
                <li>Develop a new method to detect Huanglongbing infection in citrus at as early stage as possible by detecting distinguished sugar metabolites produced by citrus leaves at the early stage of infection.</li>
                <li>Prepare leave samples for the new assay by grinding dried leaves using centrifuge and glass beads, weigh leave samples and extract metabolites from the ground leave samples using Methanol.</li>
                <li>Focus on preventing cross-contamination across individual leave samples.</li>
            </ul>
        </section>

    </section>

    <section>
        <h2>SCIENCE COMMUNICATION & OUTREACH</h2>
        
    </section>

    <section>
        <h2>PUBLICATIONS</h2>
        
    </section>

    <section>
        <h2>CONFERENCE PRESENTATIONS</h2>
        
    </section>

    <section>
        <h2>HONORS & AWARDS</h2>
        <ul>
            <li>Finalist for the Clinical Emerging Leader Award (CELA) Competition at the American Society for Nutrition 2025 Conference</li>
            <li>Henry A. Jastro Research Award for 2024-2025</li>
            <li>Henry A. Jastro Research Award for 2023-2024</li>
            <li>FASEB Conference 2021, Poster Award</li>
        </ul>
        
    </section>

    <section>
        <h2>PROFESSIONAL EXPERIENCE</h2>
        <ul>
            <li>StartupSac FastTrac Entrepreneurship Bootcamp 2025 Cohort</li>
            <li>UC Entrepreneurship Academy 2025</li>
            <li>Cancer Genomics Workshop by UCLA Institute for Quantitative & Computational Bioscience 2025</li>
            <li>Spatial Transcriptomics Workshop by UCLA Institute for Quantitative & Computational Bioscience 2025</li>
            <li>AI Institute for Food Systems (AIFS) Hackathon Competition 2023</li>
            <li>UC Entrepreneurship Academy 2022</li>
        </ul>
        
    </section>

    <section>
        <h2>SKILLS AND ABILITIES</h2>
        
    </section>

</body>
</html>